src/database/migrations/20231228184838-add-columns-to-sessions.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		await queryInterface.addColumn('sessions', 'created_by', {
+			type: Sequelize.INTEGER,
+			allowNull: true,
+		})
+		await queryInterface.addColumn('sessions', 'updated_by', {
+			type: Sequelize.INTEGER,
+			allowNull: true,
+		})
+		await queryInterface.addColumn('sessions', 'type', {
```

Comment:
We cannot set allowNull as false, it will cause issues if data exists.
```
Sequelize CLI [Node: 16.16.0, CLI: 6.6.1, ORM: 6.32.0]

Loaded configuration file "configs/postgres.js".
Using environment "development".
== 20231228184838-add-columns-to-sessions: migrating =======

ERROR: column "updated_by" of relation "sessions" contains null values
```
So I've split migration into a two step process.